megamek/src/megamek/common/BattleArmor.java

```diff
@@ -1250,8 +1250,12 @@ public int getRandomTrooper() {
                 activeTroops.add(loop);
             }
         }
-        int locInt = Compute.randomInt(activeTroops.size());
-        return activeTroops.elementAt(locInt);
+        if (!activeTroops.isEmpty()) {
+            int locInt = Compute.randomInt(activeTroops.size());
+            return activeTroops.elementAt(locInt);
```

Comment:
My IDE would complain that this `int` declaration is unnecessary.  Any reason not to just do:

`return activeTroops.elementAt(Compute.randomInt(activeTroops.size()));

?

Reply:
I was debugging an error and didnt clean this part